grading/main_configure.cpp

```diff
@@ -210,26 +210,37 @@ int main(int argc, char *argv[]) {
       	  nlohmann::json::iterator in_type = input.find("type");
       	  assert (in_type != input.end());
           assert (in_type->is_string());
-      	  assert (*in_type == "textbox" || *in_type == "codebox" || *in_type == "multiplechoice");
+      	  assert (*in_type == "short_answer" || *in_type == "codebox" || *in_type == "multiplechoice");
       	  input_obj["type"] = *in_type;
-          
+
+          // starter_value_string, optional
+          // Create a empty string
+          std::string starter_value_string = "";
+
+          // If field inside json was not empty then assign to the new string
+          auto in_starter_value_string = input.find("starter_value_string");
+          if(in_starter_value_string != input.end())
+          {
+            starter_value_string = *in_starter_value_string;
```

Comment:
We should check first here to make certain that ```in_starter_value_string``` is in fact a string, and throw an appropriate, user-readable error if it is not.